velox/functions/prestosql/aggregates/GeometricMeanAggregate.cpp

```diff
@@ -107,12 +116,14 @@ void registerGeometricMeanAggregate(const std::string& prefix) {
         if (exec::isRawInput(step)) {
           switch (inputType->kind()) {
             case TypeKind::BIGINT:
-              return std::make_unique<
-                  SimpleAggregateAdapter<GeometricMeanAggregate<int64_t>>>(
-                  resultType);
+              return std::make_unique<SimpleAggregateAdapter<
```

Comment:
With recent changes, exec::isRawInput(step) is always true. Let's remove else branch.

Reply:
`exec::isRawInput(step) is always true` is this an optimization in `SimpleAggregateAdapter`?

Reply:
done

Reply:
See https://github.com/facebookincubator/velox/pull/6934